### DISCUSSION TITLE
Download Time over 60 Minutes

### DISCUSSION THREAD

**Discussion by Aziiiz1992** [aclap-dev/video-downloadhelper#1259] [Help and Q&A]
### Your question

Hello The following question or help I want to Plex.tv an old film store but the problem is that as soon as a film is longer than 60 minutes, for example 67 then I can only load the last 7 minutes.

I can't post a link to the film because I have to register for it. Does anyone know how to get around this?

### Addon & CoApp version and details

![Screenshot 2024-09-13 090728](https://github.com/user-attachments/assets/60a46b08-46b6-4156-ae07-8f5392adf332)

**Comment by GrampaWildWilly** [comment]
You need to be posting in discussion #1258.  You started the discussion there.  Continue it there.

**Reply by Aziiiz1992** [reply]
Anstatt unnötige Kommentare abzugeben kannst du mir lieber helfen

**Reply by GrampaWildWilly** [reply]
You don't grasp the fact that I am making necessary comments.  You are not posting things that will permit me to help you.  Go back to your original thread & post there.  Follow the instructions I gave you in #1258 & post there.

**Comment by Mattmjs** [comment]
It's just an incorrect display issue for certain dash streams that are over an hour long. All it's done is left out the one.
You can be assured that you'll get the whole 1:10:53 if you download it.

**Reply by Aziiiz1992** [reply]
no it is not a false display it really only downloads everything under 1 hour does anyone know a solution

**Reply by GrampaWildWilly** [reply]
If you will continue your original discussion at #1258, and follow the instructions I gave you over there, I will try to help you.

**Comment by Mattmjs** [comment]
I tried this site which I thought I'd be geo blocked but I'm not and I
looked around a bit. I was seeing a duration of a movie over an hour in the
extension match
what it showed in the video player. This video played but Firefox indicated
to me that this site uses DRM. Sure enough I clicked on something else and
it wouldn't
play unless I enabled DRM. @Aziliz1992 I wouldn't be surprised if your
example is using DRM but your using Edge and it does not give a clue that
DRM is
running on protected content. It may or may not be the case with your
example.

I can test your example if you follow these instructions. Click the 3 dot
button next to download, look down to details and click on it.
A page will open with more information. This is the most important steps to
follow:

1. *Ctrl + A to copy all on the page.*
2. *Ctrl + C to copy information to the clipboard.*
3. *Ctrl + V to paste the information to this discussion.*

Awaiting your response but if no further response then the problem will
stay with you and won't be resolved.

**Reply by Aziiiz1992** [reply]
https://app.plex.tv/desktop/?_gl=1*rffrb7*_gcl_au*MTEzMTk5OTczOC4xNzI2MjExMDIw*_ga*Mjk1OTk3MTA0LjE3MjYyMTEwMjA.*_ga_G6FQWNSENB*MTcyNjkzNjQzNS4yLjAuMTcyNjkzNjQzNS42MC4wLjA.#!/server/e25ea7d0977e810822177de6c2223ef88240d21f/details?key=%2Flibrary%2Fmetadata%2F83411&context=source%3Ahub.movie.inprogress~0~1

The link won't work because you have to log in like with Netflix etc.

it doesn't work, I can only download anything under an hour (under 60 minutes) if I download something that takes, for example, 1 hour 32 minutes, only the last 32 minutes are downloaded

**Reply by Mattmjs** [reply]
This link isn't of any use to me, even if it requires log in. I click it and the page says not found. It must only be accessible for
German viewers. Why didn't you provide what I asked for ? I gave you some directions to follow that would offer more information
about the media. I need that information if you want help. Do you still want help or not.
![Screenshot 2024-09-23 at 16-57-13 Plex](https://github.com/user-attachments/assets/7f5def20-0e7b-4d31-83d0-e4e5eb1cdeb8)

**Reply by Mattmjs** [reply]
![16-7-24-22 00](https://github.com/user-attachments/assets/3320cf6f-047d-4fa4-a8cd-027e8289bfd6)
